interview-questions.md

```diff
@@ -9,41 +9,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: In my experience 'this' in JavaScript is used to call on a key value in a class.
 
-  Researched answer:
+  Researched answer: 'this' is a JavaScript keyword that refers to the object it belongs to.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a library in JavaScript and is used in front end development. From my understanding we use react to make applications more dynamic.
 
-  Researched answer:
+  Researched answer: React is JavaScript library for creating user interfaces. We use it to build small parts and then join them together to form entire applications. 
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: From my understanding render is required in a react class component 
 
-  Researched answer:
+  Researched answer: The render is required because it returns the html element.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: In my experience JSX is used to bring in JavaScript inside a html. The difference between HTML and JSX is the way the class is named.
 
-  Researched answer:
+  Researched answer: JSX is JavaScript flavored HTML that acts almost like HTML. The most notably difference JSX uses the syntax className rather than HTML's class because class is reserved for HTML.
```

Comment:
Great!  Another important difference is that JSX renders only one element which is why we need react fragments when adding multiple elements.